dev-docs/bidders/consumable-server.md

```diff
@@ -0,0 +1,373 @@
+---
+layout: bidder
+title: Consumable
+description: Prebid Consumable Bidder adapter
+userIds: all
+pbjs: true
+pbs: true
+pbs_app_supported: true
+schain_supported: true
+coppa_supported: true
+tcfeu_supported: true
+floors_supported: true
+usp_supported: true
+gpp_supported: true
+biddercode: consumable
+media_types: banner, video, audio
```

Comment:
Please specify

* multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid
* safeframes_ok: true/false

if you support multiple formats